api/kv.go

```diff
@@ -50,21 +50,21 @@ type KVOp string
 
 const (
 	KVSet          KVOp = "set"
-	KVDelete            = "delete"
-	KVDeleteCAS         = "delete-cas"
-	KVDeleteTree        = "delete-tree"
-	KVCAS               = "cas"
-	KVLock              = "lock"
-	KVUnlock            = "unlock"
-	KVGet               = "get"
-	KVGetTree           = "get-tree"
-	KVCheckSession      = "check-session"
-	KVCheckIndex        = "check-index"
+	KVDelete       KVOp = "delete"
```

Comment:
afaik that only works for `iota` assignments. This is the output of gitst https://gist.github.com/alicebob/aa17022b6f1d6f8fbf3337e2c178c858

```
KVSet: main.KVOp
KVDelete: main.KVOp
KVDeleteCAS: string
KVDeleteTree: string

```